packages/graphql/src/translate/authorization/compatibility/compile-predicate-return.ts

```diff
@@ -35,7 +37,8 @@ type CompiledPredicateReturn = {
  */
 export function compilePredicateReturn(
     predicateReturn: PredicateReturn,
-    indexPrefix?: string
+    indexPrefix: string | undefined,
+    context: Neo4jGraphQLTranslationContext
 ): CompiledPredicateReturn {
```

Comment:
I think this might be nicer using an object for the params mostly for readability in the function calls

```ts
export function compilePredicateReturn({
    predicateReturn,
    indexPrefix,
    context
}: {
    predicateReturn: PredicateReturn;
    indexPrefix?: string;
    context: Neo4jGraphQLTranslationContext;
})
```

and then for example:

```ts
return compilePredicateReturn({ predicateReturn, indexPrefix: `${indexPrefix || "_"}after_`, context });
```